lib/hooks.go

```diff
@@ -30,9 +30,16 @@ type HookParams struct {
 	Arguments     []string `json:"arguments"`
 }
 
+var (
+	errNotJSON = errors.New("hook file isn't a JSON")
```

Comment:
I'd rather avoid contractions in error messages.  How about “hook path does not end in .json”?